compiler/ballerina-lang/src/main/java/io/ballerina/projects/configurations/ConfigModuleDetails.java

```diff
@@ -27,12 +29,15 @@
     String packageName;
     String moduleName;
     boolean isCurrentPackage;
+    ModuleDescriptor moduleDescriptor;
 
-    public ConfigModuleDetails(String orgName, String packageName, String moduleName, boolean isCurrentPackage) {
+    public ConfigModuleDetails(String orgName, String packageName, String moduleName, boolean isCurrentPackage,
```

Comment:
ModuleDescriptor contains the PacakgeDescriptor which contains the org and and name of the package. So we can remove the `orgName` and `packageName` params